package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/ExportHelpers.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.IO;
+using UnityEditor;
+using UnityEngine;
+using USD.NET;
+
+namespace Unity.Formats.USD
+{
+    public static class ExportHelpers
+    {
+#if UNITY_EDITOR
+        public static Scene InitForSave(string defaultName, string fileExtension = "usd,usda,usdc")
+        {
+            var filePath = EditorUtility.SaveFilePanel("Export USD File", "", defaultName, fileExtension);
+
+            if (filePath == null || filePath.Length == 0)
+            {
+                return null;
+            }
+
+            var fileDir = Path.GetDirectoryName(filePath);
+
+            if (!Directory.Exists(fileDir))
+            {
+                var di = Directory.CreateDirectory(fileDir);
+                if (!di.Exists)
+                {
+                    Debug.LogError("Failed to create directory: " + fileDir);
+                    return null;
+                }
+            }
+
+            InitUsd.Initialize();
+            var scene = Scene.Create(filePath);
+            scene.Time = 0;
+            scene.StartTime = 0;
+            scene.EndTime = 0;
+            return scene;
+        }
+#endif
+
+        public static void ExportSelected(BasisTransformation basisTransform,
+            string fileExtension = "usd,usda,usdc",
+            bool exportMonoBehaviours = false)
+        {
+            Scene scene = null;
+
+            foreach (GameObject go in Selection.gameObjects)
```

Comment:
Selection exists only in the editor. Make a function that exports a list of GOs and call that with the selection from the editor assembly.

Reply:
Fixed in 3cd29eb4